app/components-react/shared/PlatformAppPageView.tsx

```diff
@@ -102,7 +102,13 @@ export default function PlatformAppPageView(p: {
         </div>
       )}
       <div
-        style={{ display: 'flex', flexDirection: 'column', width: '100%', ...p.style }}
+        className="platform-app"
+        style={{
+          display: !hideStyleBlockers ? 'flex' : 'none',
```

Comment:
this seems leftover?

Reply:
The individual apps are rendered differently so the whole component needs to be hidden and shown. I'll remove the className though